sos/policies/distros/__init__.py

```diff
@@ -691,11 +691,7 @@ def upload_sftp(self, user=None, password=None):
         # soft dependency on python3-pexpect, which we need to use to control
         # sftp login since as of this writing we don't have a viable solution
         # via ssh python bindings commonly available among downstreams
-        try:
-            import pexpect
-        except ImportError:
-            raise Exception('SFTP upload requires python3-pexpect, which is '
-                            'not currently installed')
+        import pexpect
```

Comment:
When `python3-pexpect` is not installed, this will lead to message

    Upload attempt failed: No module named 'pexpect'

that does not tell the user how to fix it. Could the original error/exception text be preserved?

Reply:
ack, I've put it back now. Thanks.